execution/executor/src/lib.rs

```diff
@@ -513,13 +518,20 @@ impl<V: VMExecutor> ChunkExecutor for Executor<V> {
         // 1. Update the cache in executor to be consistent with latest synced state.
         self.reset_cache()?;
 
-        info!(
-            "Local synced version: {}. First transaction version in request: {:?}. \
-             Number of transactions in request: {}.",
-            self.cache.synced_trees().txn_accumulator().num_leaves() - 1,
-            txn_list_with_proof.first_transaction_version,
-            txn_list_with_proof.transactions.len(),
-        );
+        send_struct_log!(executor_log(LogEntry::Chunk)
```

Comment:
Could you use the log leveled macros?  `sl_info!` `sl_debug!` etc.

Trying to have all logs have levels :)